exporter/datadogexporter/internal/metrics/system.go

```diff
@@ -130,6 +147,20 @@ func PrepareSystemMetrics(ms []datadog.Metric) []datadog.Metric {
 		// all existing system metrics need to be prepended
 		newname := otelNamespacePrefix + *m.Metric
 		series[i].Metric = &newname
+
+		// Map metric to equivalent Datadog runtime metric if found
+		var ddruntimename string
+		for otelname, ddname := range goRuntimeMetricsMappings {
+			if *m.Metric == otelname {
+				ddruntimename = ddname
+				break
+			}
+		}
+		if ddruntimename != "" {
+			ddruntime := series[i]
+			ddruntime.Metric = &ddruntimename
+			ddruntimemetrics = append(ddruntimemetrics, ddruntime)
+		}
```

Comment:
```suggestion
		if v, ok := goRuntimeMetricsMappings[*m.Metric]; ok {
			// remap runtime metric to Datadog convention
			m.Metric = &v
			series = append(series, m)
		}
```
This should be all you need to do. `m` is already a copy of `series[i]` (which is the original), so you can simply append it at this point and undo the other changes.

Make sure to add tests too.